pkg/types/time_test.go

```diff
@@ -1814,7 +1814,8 @@ func TestParseTimeFromFloat64(t *testing.T) {
 		{0.0, mysql.TypeDate, 0, 0, 0, 0, 0, 0, 0, nil},
 		{20000102030405, mysql.TypeDatetime, 2000, 1, 2, 3, 4, 5, 0, nil},
 		{20000102030405.015625, mysql.TypeDatetime, 2000, 1, 2, 3, 4, 5, 15625, nil},
-		{20000102030405.0078125, mysql.TypeDatetime, 2000, 1, 2, 3, 4, 5, 7812, nil},
+		{20000102030405.0078125, mysql.TypeDatetime, 2000, 1, 2, 3, 4, 5, 7813, nil},
```

Comment:
The significant digits of the float fractional part is only '.008' I guess, so does it really matter if it is '.007812' or '.007813' ?
Just try this with your PR:
```
{20000102030405.0078139, mysql.TypeDatetime, 2000, 1, 2, 3, 4, 5, 7814, nil},
```

Reply:
Yes. The fractional part is only `.008` so the fractional part of your case is still `7813`.
As https://github.com/pingcap/tidb/issues/56339#issuecomment-2401552101 said, I just want to make this behavior consistent with TiKV and MySQL.

Reply:
OK, then I see.
Can you please add tests that will test TiKV/TiFlash/UniStore handling of co-processor calls for this (if exists), so we can show that all four implementations are the same?

Reply:
@mjonss is this now resolved?

Reply:
@gengliqi I only see a single integration test. Are you sure it is testing with both TiKV and Unistore?

Reply:
Yes, the `check_dev` test runs these integration tests with Unistore while the `check_dev_2` runs with a real TiKV.